money-core/src/test/scala/com/comcast/money/emitters/LogFormatterSpec.scala

```diff
@@ -0,0 +1,156 @@
+      )
+    }
+    "honor key names from the config" in {
+      val conf = ConfigFactory.parseString(
+        """
+              {
+                emitter="com.comcast.money.emitters.LogRecorder"
+                formatting {
+                  keys {
+                    span-id = "spanId"
+                    trace-id = "traceId"
+                    parent-id = "parentId"
+                    span-name = "spanName"
+                    app-name = "appName"
+                    start-time = "startTime"
+                    span-duration = "spanDuration"
+                    span-success = "spanSuccess"
+                  }
+                }
+              }
+        """
+      )
+      val spanLogFormatter = SpanLogFormatter(conf)
+
+      val sampleData = Span(
+        SpanId(1L), "key", "unknown", "host", 1L, true, 35L,
+        Map("what" -> Note("what", 1L), "when" -> Note("when", 2L), "bob" -> Note("bob", "craig"))
+      )
+      val actualMessage = spanLogFormatter.buildMessage(sampleData)
+      assert(
+        actualMessage === ("Span: [ spanId=1 ][ traceId=1 ][ parentId=1 ][ spanName=key ][ appName=unknown ][ " +
+          "startTime=1 ][ spanDuration=35 ][ spanSuccess=true ][ bob=craig ][ what=1 ][ when=2 ]")
+      )
+    }
+    "honor the span-start from the config" in {
+      val conf = ConfigFactory.parseString(
+        """
+              {
+                emitter="com.comcast.money.emitters.LogRecorder"
+                formatting {
+                  span-start = "Start :|: "
+                }
+              }
+        """
+      )
+      val spanLogFormatter = SpanLogFormatter(conf)
+      val sampleData = Span(
+        SpanId(1L), "key", "unknown", "host", 1L, true, 35L,
+        Map("what" -> Note("what", 1L), "when" -> Note("when", 2L), "bob" -> Note("bob", "craig"))
+      )
+      val actualMessage = spanLogFormatter.buildMessage(sampleData)
+      assert(
+        actualMessage === ("Start :|: [ span-id=1 ][ trace-id=1 ][ parent-id=1 ][ span-name=key ][ app-name=unknown ][ " +
+          "start-time=1 ][ span-duration=35 ][ span-success=true ][ bob=craig ][ what=1 ][ when=2 ]")
+      )
+    }
+    "honor the log-template from the config" in {
+      val conf = ConfigFactory.parseString(
+        """
+              {
+                emitter="com.comcast.money.emitters.LogRecorder"
+                formatting {
+                  log-template = "%s=\"%s\" "
+                }
+              }
+        """
+      )
+      val spanLogFormatter = SpanLogFormatter(conf)
+      val sampleData = Span(
+        SpanId(1L), "key", "unknown", "host", 1L, true, 35L,
+        Map("what" -> Note("what", 1L), "when" -> Note("when", 2L), "bob" -> Note("bob", "craig"))
+      )
+      val actualMessage = spanLogFormatter.buildMessage(sampleData)
+      assert(
+        actualMessage === ("Span: span-id=\"1\" trace-id=\"1\" parent-id=\"1\" span-name=\"key\" " +
```

Comment:
Instead of escaping the quotes here, you could use the triple quote I think, then ditch those escape characters.

```scala
actualMessage === """
```